include/verbose.mk

```diff
@@ -29,15 +29,15 @@ ifeq ($(IS_TTY),1)
   endif
 endif
 
-define ERROR_MESSAGE
-  printf "$(_R)%s$(_N)\n" "$(1)" >&8
-endef
-
 ifeq ($(findstring s,$(OPENWRT_VERBOSE)),)
   define MESSAGE
 	printf "$(_Y)%s$(_N)\n" "$(1)" >&8
   endef
 
+  define ERROR_MESSAGE
+	printf "$(_R)%s$(_N)\n" "$(1)" >&8
+  endef
```

Comment:
I think we should move this pat to #14039.

Reply:
my PR is meant to solve that issue without needing this revert patch

Reply:
Yes, I'll leave this for Michael and Paul to dive into. Main thing is that it does get resolved...